library-no-op/src/main/kotlin/com/chuckerteam/chucker/api/ExportFormat.kt

```diff
@@ -0,0 +1,6 @@
+package com.chuckerteam.chucker.api
+
+public enum class ExportFormat {
+    LOG,
+    HAR
```

Comment:
let's add a field inside this enum called `extension: String` which we can use to get the file extension